### ISSUE TITLE
newrelic_synthetics_multilocation_alert_condition: violation_time_limit_seconds required or not?  Bug documentation?

### ISSUE THREAD

**Issue by xmariopereira** [newrelic/terraform-provider-newrelic#1519]
Documentation: https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/synthetics_multilocation_alert_condition

```
Argument Reference
The following arguments are supported:

name - (Required) The title of the condition.
policy_id - (Required) The ID of the policy where this condition will be used.
runbook_url - (Optional) Runbook URL to display in notifications.
enabled - (Optional) Set whether to enable the alert condition. Defaults to true.
entities - (Required) The GUIDs of the Synthetics monitors to alert on.
critical - (Required) A condition term with the priority set to critical.
warning - (Optional) A condition term with the priority set to warning.
````

Error:
```
 terraform validate
╷
│ Error: Missing required argument
│ 
│   on modules/synthetics-monitor/main.tf line 6, in resource "newrelic_synthetics_multilocation_alert_condition" "synthetics_alert_status_page":
│    6: resource "newrelic_synthetics_multilocation_alert_condition" "synthetics_alert_status_page" {
│ 
│ The argument "violation_time_limit_seconds" is required, but no definition was found.
```

Go code (TF):
```
			"violation_time_limit_seconds": {
				Type:         schema.TypeInt,
				Required:     true,
				ValidateFunc: validation.IntInSlice([]int{0, 3600, 7200, 14400, 28800, 43200, 86400}),
				Description:  "The maximum number of seconds a violation can remain open before being closed by the system.  Must be one of: 0, 3600, 7200, 14400, 28800, 43200, 86400",
			},
```
Go code (Go-client)

```
type MultiLocationSyntheticsCondition struct {
	ID                        int                                    `json:"id,omitempty"`
	Name                      string                                 `json:"name,omitempty"`
	Enabled                   bool                                   `json:"enabled"`
	RunbookURL                string                                 `json:"runbook_url,omitempty"`
	Entities                  []string                               `json:"entities,omitempty"`
	Terms                     []MultiLocationSyntheticsConditionTerm `json:"terms,omitempty"`
	ViolationTimeLimitSeconds int                                    `json:"violation_time_limit_seconds,omitempty"`
}
```

In newrelic_nrql_alert_condition, the field violation_time_limit_seconds is not required

https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/nrql_alert_condition

```
violation_time_limit_seconds - (Optional) Sets a time limit, in seconds, that will automatically force-close a long-lasting violation after the time limit you select. The value must be between 300 seconds (5 minutes) to 2592000 seconds (30 days) (inclusive).
*Note: One of violation_time_limit or violation_time_limit_seconds must be set, but not both.
```

Why not making synthetics_multilocation_alert_condition and newrelic_nrql_alert_condition equal?

Even the maximum value is different in both... one is 2592000, and the other one is 86400...

**Comment by mbazhlekova** [comment]
@xmariopereira Good catch! This looks like an oversight in the documentation.

**Comment by xmariopereira** [comment]
fixed in #1521
